### PR TITLE
The Asynchrnous Javascript  git curse notes and learning

### PR DESCRIPTION
**Pull request by tom-ramirez**
The Asynchrronous javascript course

### CODE REVIEW

File: .idea/misc.xml
Comment (alexeikmun) [inline .idea/misc.xml:1]: Please remove these files from your staging by using a `.gitignore` file

File: .idea/modules.xml
Comment (alexeikmun) [inline .idea/modules.xml:1]: same

File: .idea/tomas-ramirez.iml
Comment (alexeikmun) [inline .idea/tomas-ramirez.iml:1]: same

File: .idea/vcs.xml
Comment (alexeikmun) [inline .idea/vcs.xml:1]: same

File: AdvancedGit/AdvancedGit.iml
Comment (alexeikmun) [inline AdvancedGit/AdvancedGit.iml:1]: same

**Review by alexeikmun — CHANGES_REQUESTED** [review]

File: .idea/modules.xml
Comment (tom-ramirez) [reply .idea/modules.xml:1]: working on it

**Review by alexeikmun — APPROVED** [review]

**Pull request merged**